### PR TITLE
Armory authorization computer

### PR DESCRIPTION
**Pull request by mybluecorners**
## About the PR 
This allows members of security to vote together to open up the Armory.
3 votes are required from unique crewmembers (no gaming the system with multiple IDs) to open Armory access. A single HoS vote can open things up as well.
This isn't actually added to any maps yet - i will do that after merging

### CODE REVIEW

**Comment by ryebek** [comment]
Should probably also open up the crates/lower their access requirements, if that's even possible, as well as reduce the ammovend access requirement, as those have a lot of good and useful stuff.

**Comment by mybluecorners** [comment]
> Should probably also open up the crates/lower their access requirements, if that's even possible, as well as reduce the ammovend access requirement, as those have a lot of good and useful stuff.

already in the PR!

File: code/obj/item/clothing/armor.dm [hunk 1]
```diff
@@ -314,7 +314,7 @@
 		setProperty("meleeprot", 12)
 		setProperty("rangedprot", 3)
 		setProperty("pierceprot",25)
-		setProperty("disorient_resist", 25)
+		setProperty("disorient_resist", 45)
```
Comment (Tarmunora) [inline code/obj/item/clothing/armor.dm:317]: sneaky balance change I see...

File: code/obj/machinery/computer/security/riot.dm [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+		if("Authorize")
+			if (user in src.authorized)
+				boutput(user, "You have already authorized! [src.auth_need - src.authorized.len] authorizations from others are still needed.")
+				return
+			if (access_maxsec in W:access)
+				authorize()
+				src.authorized = null
+				return
+
+			src.authorized += user //authorize by USER, not by registered ID. prevent the captain from printing out 3 unique ID cards and getting in by themselves.
```
Comment (Tarmunora) [inline code/obj/machinery/computer/security/riot.dm:106]: Might be interesting if it went by DNA, so a changeling, after eating three officers, could open the armory. Also this seems to imply that three staffies with a single stolen ID can pop the armory open?

Comment (mybluecorners) [reply code/obj/machinery/computer/security/riot.dm:106]: lol oh yeah sharing an id - i'll have to fix this

**Comment by mybluecorners** [comment]
> Changelog

this changes nothing about the gameplay in itself, the changelog will come when i actually add to maps

**Comment by Tarmunora** [comment]
The riot armor change?

**Pull request merged**